examples/cymbal-bank/README.md

```diff
@@ -82,41 +82,594 @@ The steps below assume that you are checkout out on the same level as `terraform
 1. Copy the `examples/cymbal-bank/4-appfactory` folder content to the repo:
 
     ```bash
-    cp -R terraform-google-enterprise-application/examples/cymbal-bank/4-appfactory/* eab-applicationfactory/4-appfactory/
+    mkdir ./eab-applicationfactory/envs/
+    cp -R ./terraform-google-enterprise-application/examples/cymbal-bank/4-appfactory/envs/* ./eab-applicationfactory/envs/
+    cp -R ./terraform-google-enterprise-application/4-appfactory/modules/* ./eab-applicationfactory/modules/
     ```
+1. Use `terraform output` to get the state bucket value from 1-bootstrap output and replace the placeholder in `terraform.tfvars`.
+
+   ```bash
+   cd eab-applicationfactory
+   terraform -chdir="../terraform-google-enterprise-application/1-bootstrap/" init
+   export remote_state_bucket=$(terraform -chdir="../terraform-google-enterprise-application/1-bootstrap/" output -raw state_bucket)
+
+   echo "remote_state_bucket = ${remote_state_bucket}"
+
+   sed -i'' -e "s/REMOTE_STATE_BUCKET/${remote_state_bucket}/" ./terraform.tfvars
+
+   sed -i'' -e "s/UPDATE_ME/${remote_state_bucket}/" ./envs/shared/backend.tf
+   ```
 
 1. Navigate to Application Factory repository and checkout plan branch:
 
     ```bash
-    cd eab-applicationfactory
     git checkout plan
     git add .
     git commit -m 'Adds Cymbal bank code'
     git push --set-upstream origin plan
     ```
 
-1. Merge changes to development. Because this is a named environment branch, pushing to this branch triggers both terraform plan and terraform apply. Review the apply output in your Cloud Build project https://console.cloud.google.com/cloud-build/builds;region=DEFAULT_REGION?project=YOUR_CLOUD_BUILD_PROJECT_ID
+1. Merge changes to production. Because this is a named environment branch, pushing to this branch triggers both terraform plan and terraform apply. Review the apply output in your Cloud Build project https://console.cloud.google.com/cloud-build/builds;region=DEFAULT_REGION?project=YOUR_CLOUD_BUILD_PROJECT_ID
+
+    ```bash
+    git checkout production
+    git merge plan
+    git push origin production
+    ```
+
+1. Move out of App Factory folder:
+
+    ```bash
+    cd ../
+    ```
+
+#### Add Cymbal Bank envs at App Infra
+
+1. Retrieve Cymbal Bank repositories created on 4-appfactory.
+
+    ```bash
+    cd eab-applicationfactory/envs/shared/
+    terraform init
+
+    export balancereader_project=$(terraform output -json app-group | jq -r '.["cymbal-bank.balancereader"]["app_admin_project_id"]')
```

Comment:
Just a thought, if `app_admin_project_id`, `app_infra_repository_name`, and `app_cloudbuild_workspace_state_bucket_name` were output in an `app` (or similar name) **array**, they could instead be passed in a single command.